debian/changelog

```diff
@@ -1,10 +1,35 @@
-cloud-init (23.3-0ubuntu0~23.04.1) UNRELEASED; urgency=medium
+cloud-init (23.4-0ubuntu0~23.04.1) lunar; urgency=medium
```

Comment:
Since this is a backport from devel, this should be `23.4-0ubuntu1~23.04.1`. Once current branches merge, I'd like to update our tooling accordingly (if possible). https://github.com/canonical/ubuntu-maintainers-handbook/blob/main/VersionStrings.md#version-backport-from-ubuntu-devel